### ISSUE TITLE
[Question] Adding more controls to a formgroup after form initialization

### ISSUE THREAD

**Issue by Ellipizle** [udos86/ng2-dynamic-forms#252]
I am trying to add more form control to a formgroup after initialization of a form, but i am getting this error
```javascript
TypeError: Cannot read property 'valueChanges' of null
```
This is how i am doing it currrently
```javascript
export const MY_DYNAMIC_FORM_MODEL: Array<DynamicFormControlModel> = [
          new DynamicInputModel({

                id: "first input",
                label: "Example first input",
                value: "Test 1-2"
            });
    new DynamicFormGroupModel({

        id: "basicFormGroup1",
        legend: "Form Group 1",
        group: []
    }),
];  
 // initialize the form
 this.formGroup = this.formService.createFormGroup(this.formModel);

 }
```

```javascript

//This method finds a dynamicFormGroupModel with the given id and add a model(e.g DynamicInputModel) based on the controltype
  addChild(groupid, controltype) {
    //find group of the given id
    let group = this.findGroup(this.formModel, groupid)
    console.log(group);
   
 //add a form control to the group
    this.addControlToGroup(controltype,group.group)

  //the new model is added to the formgroup
    console.log(this.formModel)
 
 //this throws error Cannot read property 'valueChanges' of null
    this.formGroup = this.dynamicFormService.createFormGroup(this.formModel);
  }
```

**Comment by udos86** [comment]
@Ellipizle Thanks for your message! Please give me a bit of time to investigate your problem! I'll let you know.

**Comment by udos86** [comment]
@Ellipizle I'll try to deliver service functions to add and remove form controls in form groups in the next release

**Comment by udos86** [comment]
@Ellipizle Please have a look at 801a3aa

**Comment by Ellipizle** [comment]
Thank you very much for your help

**Comment by udos86** [comment]
@Ellipizle No problem! Will be released in `1.3.9`.
